thejamesgore/Foxtons-Dashboard
# feat: Add all-properties page with PropertyCard component with light/mode styling

## Description 🚀
This pull request adds the `all-properties.tsx` page and `PropertyCard.tsx` component to the project, with consistent light and dark mode styling.

## Changes Made 👉
- Created `all-properties.tsx` page to render all properties in the dashboard using the `PropertyCard.tsx` component
- Added `PropertyCard.tsx` component to `all-properties.tsx` page
- Implemented consistent light and dark mode styling using the `ColorModeContext`

## Reason for Changes 🔍
The changes were made to provide a comprehensive view of all properties in the dashboard and improve the user experience by adding a new component and consistent styling.

## How to Test
To test these changes, you can follow these steps:
- Clone this branch to your local machine
- Install the necessary dependencies with `npm install`
- Start the application with `npm start`
- Navigate to the `all-properties.tsx` page
- Verify that all properties are displayed correctly using the `PropertyCard` component
- Verify that the styling is consistent with the theme in both light and dark mode

## Checklist

## Related Issues
None.

## Other Notes
None.
